scripts/compile_proto.sh

```diff
@@ -7,7 +7,7 @@ mkdir -p proto/js
 for filepath in proto/*.proto; do
   filename=$(basename "$filepath" .proto)
   # Compiles JavaScript
-  pbjs -t static-module -w commonjs --no-beautify --no-convert --no-verify --no-delimited -o proto/js/$filename.js $filepath
+  pbjs -t static-module -w commonjs --no-beautify --no-convert --no-verify -o proto/js/$filename.js $filepath
 
   # Compiles type definition files from JavaScript files
   pbts -o proto/js/$filename.d.ts proto/js/$filename.js -n $filename
```

Comment:
What's the difference between `pbjs` and `pbts`?

Reply:
pbjs is protobufjs cli and pbts is for typescript which generated the `.d.ts` files